api-controllers/ServiceFabrikApiController.js

```diff
@@ -450,7 +450,7 @@ class ServiceFabrikApiController extends FabrikBaseController {
         }
```

Comment:
What does it mean to send timestamp in a request for non-pitr service? Should that be also validated before proceeding?

Reply:
Fabrik searches for a base backup before the requested timestamp. while triggering the backup via agent api backup_guid also sent in this case. Transaction log replaying happens in case time_stamp is also present. 

The check need not to be implemented at server end. If required can be introduced in client (cf cli/ UI based).
@jagadish-kb  @kamath-prasad  What is your opinion?

Reply:
I think this validation is not necessary since anyways agent will not consume the timestamp even if passed.

Reply:
added a check for non pitr